src/main/java/com/google/api/codegen/metacode/InitCodeNode.java

```diff
@@ -157,12 +157,13 @@ public static InitCodeNode createSingletonList(String key) {
     return InitCodeNode.createWithChildren(key, InitCodeLineType.ListInitLine, child);
   }
 
+  /** Creates a node to be used as the root of the initialization tree. */
   public static InitCodeNode newRoot() {
     return new InitCodeNode("root", InitCodeLineType.StructureInitLine, InitValueConfig.create());
   }
 
   /*
-   * Constructs a tree of objects to be initialized using the provided context, and returns the root
+   * Constructs a tree of objects to be initialized using the provided context, and returns the root.
```

Comment:
Make into a one-line java doc (it's not a javadoc now)
`/** ... */`

Reply:
Made into Javadoc, though I think the formatter wants to keep them 3 lined.